packages/next/src/client/components/react-dev-overlay/app/hot-reloader-client.tsx

```diff
@@ -501,13 +516,24 @@ function processMessage(
       // TODO-APP: potentially only refresh if the currently viewed page was added/removed.
       return router.hmrRefresh()
     }
+    case HMR_ACTIONS_SENT_TO_BROWSER.AFTER_ERROR: {
```

Comment:
I was assuming we just `console.error` these. That way we cover Terminal, Node.js debuggers and the error overlay.

Reply:
as discussed offline, the problem is that `after` errors occur after everything else, so we can't rely on e.g. react's error replaying to get them here, we need to push them (technically the client could also try to pull, maybe that's a better strategy?)